traffic_ops/traffic_ops_golang/deliveryservice/acme.go

```diff
@@ -159,7 +159,8 @@ func GenerateAcmeCertificates(w http.ResponseWriter, r *http.Request) {
 		api.HandleErr(w, r, inf.Tx.Tx, http.StatusInternalServerError, nil, errors.New("deliveryservice.GenerateAcmeCertificates: Traffic Vault is not configured"))
 		return
 	}
-	ctx, _ := context.WithTimeout(r.Context(), AcmeTimeout)
+	ctx, cancelTx := context.WithTimeout(r.Context(), AcmeTimeout)
+	defer cancelTx()
```

Comment:
I'm not sure this is what we want, since this will cancel the context before the spawned goroutine on L208 (which uses the context) has a chance to complete its task.

Reply:
Avoiding cancelling the context before it is used in e55a35c622.